storage/client/src/main/java/com/azure/storage/blob/BlobAsyncRawClient.java

```diff
@@ -255,32 +225,31 @@
      * For more samples, please see the [Samples file](%https://github.com/Azure/azure-storage-java/blob/master/src/test/java/com/microsoft/azure/storage/Samples.java)
      */
     public Mono<DownloadAsyncResponse> download(BlobRange range, BlobAccessConditions accessConditions,
-                                             boolean rangeGetContentMD5, Context context) {
-        Boolean getMD5 = rangeGetContentMD5 ? rangeGetContentMD5 : null;
+                                             boolean rangeGetContentMD5) {
```

Comment:
Shouldn't downloadToFile also be on the raw client?

Reply:
The raw client isn't exposed to users.